### ISSUE TITLE
Moving all media to central

### ISSUE THREAD

**Issue by pixeldevsio** [bueltge/multisite-global-media#112]
Any idea how to move existing media to the central site? Maybe through CLI (or really any way possible)

**Comment by widoz** [comment]
I think you can use this https://developer.wordpress.org/cli/commands/media/import/ then using https://developer.wordpress.org/cli/commands/post/delete/ to delete the posts.

**Comment by pixeldevsio** [comment]
WP Media Import would work, but it doesn’t keep connections to existing posts / ACF fields / anything else. It creates a new instance of the attachment with a new ID

**Comment by widoz** [comment]
> WP Media Import would work, but it doesn’t keep connections to existing posts / ACF fields / anything else. It creates a new instance of the attachment with a new ID

Correct, that case you have to create a custom script to update the db using wp cli internally. 
Honestly I don't know, could be an interesting feature but right now appear to be out of the scope of the plugin.
